Applications/Resnet/jni/main.cpp

```diff
@@ -15,6 +15,70 @@
 #include <vector>
 
 #include <layer.h>
+#include <model.h>
+
+using LayerHandle = std::shared_ptr<ml::train::Layer>;
+using ModelHandle = std::unique_ptr<ml::train::Model>;
+
+/**
+ * @brief make "key=value" from key and value
+ *
+ * @tparam T type of a value
+ * @param key key
+ * @param value value
+ * @return std::string with "key=value"
+ */
+template <typename T>
+static std::string withKey(const std::string &key, const T &value) {
+  std::stringstream ss;
+  ss << "key=" << value;
+  return ss.str();
+}
+
+template <typename T>
+static std::string withKey(const std::string &key,
+                           const std::vector<T> &value) {
+  if (value.empty()) {
+    throw std::invalid_argument("empty vector cannot be converted");
+  }
+
+  std::stringstream ss;
+  ss << "key=";
+  for (unsigned int i = 0; i < value.size() - 1; ++i) {
+    ss << value.at(i) << ',';
```

Comment:
maybe `,` -> `:` (i am thinking this is for dimension)?

Reply:
This will be for the kernelsize, poolsize those kind of stuff

Reply:
alright.